src/engine/ScreenFilter.cs

```diff
@@ -2,38 +2,67 @@
 
 /// <summary>
 ///   Handles what colorblind filter is used
-///   Turns off filter if set to "normal_color"
 /// </summary>
 public class ScreenFilter : TextureRect
 {
+    private static ScreenFilter instance;
     private ShaderMaterial screenFilterMaterial;
+    public ScreenFilter()
```

Comment:
add a blank line before this

Reply:
also this should be private like GUICommon to avoid someone trying to create a new instance of this class.